projects/js-packages/idc/changelog/update-idc-stories

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
IMO I don't think this change deserves `minor` significance. Using `patch` should free you up from bumping dependencies.